serving/docker/scripts/install_inferentia2.sh

```diff
@@ -15,12 +14,9 @@ echo "deb https://apt.repos.neuron.amazonaws.com ${VERSION_CODENAME} main" >/etc
 curl -L https://apt.repos.neuron.amazonaws.com/GPG-PUB-KEY-AMAZON-AWS-NEURON.PUB | apt-key add -
 
 # https://awsdocs-neuron.readthedocs-hosted.com/en/latest/release-notes/releasecontent.html#inf2-packages
-apt-get update -y && apt-get install -y linux-headers-$(uname -r) && apt-get install -y aws-neuronx-dkms=2.8.4.* \
-    aws-neuronx-collectives=2.12.27.* \
-    aws-neuronx-runtime-lib=2.12.16.* \
-    aws-neuronx-tools=2.9.5.*
+apt-get update -y && apt-get install -y linux-headers-$(uname -r) && apt-get install -y aws-neuronx-dkms=2.9.* \
```

Comment:
NIT: The release [page](https://awsdocs-neuron.readthedocs-hosted.com/en/latest/release-notes/releasecontent.html#inf2-packages) lists `dkms==2.9.1` along-with versions `collectives==2.12, runtime-lib==2.12`. Do we want to keep them in sync?